### ISSUE TITLE
html-sanitizer mixing property names in arrays causing headaches for Closure Compiler

### ISSUE THREAD

**Issue by mikesamuel** [google/caja#1989]
Kaitlyn E would like some errors from Closure Compiler to go away:

.../html-sanitizer-bundle.js:2078: ERROR - restricted index type
found   : `string`
required: `number`

```js
          attribs = decision['attribs'];
                             ^
```

html-sanitizer-bundle.js:2085: ERROR - restricted index type
found   : `string`
required: `number`

```js
          tagNameRep = decision['tagName'];
                                ^
```

**Comment by mikesamuel** [comment]
https://github.com/google/caja/blob/110311969a9e3d8acfe38526fe9d925485648fc8/src/com/google/caja/plugin/html-sanitizer.js#L690,L698 are the relevant lines.

**Comment by mikesamuel** [comment]
Outstanding pull request https://github.com/google/caja/pull/1981/files may actually address this.

**Comment by kpreid** [comment]
Fixed with 27248d0f223038346d63278b52ec038e77418ba9
